### ISSUE TITLE
row_loss not going down

### ISSUE THREAD

**Issue by sudokhan112** [aliyun/conditional-lane-detection#27]
I am training culane_small with only two images to overfit. It's a crop-row image.  But the row_loss is not going down. It's around 22. Is this normal? How can I tune to reduce the loss? Also after training, if I run culane_test, test output has no line drawn on them. Here is a ground truth image:

![0_d03caef42cc4688a2ee02217f26c46e 00000 jpg gt](https://user-images.githubusercontent.com/77024625/131152598-66295049-5fa5-49aa-8b82-fef646f25748.jpg)
